tests/drivers/pwm/gpio_loopback/testcase.yaml

```diff
@@ -0,0 +1,14 @@
+tests:
+  drivers.pwm.loopback:
+    tags:
+      - pwm
+      - drivers
+    depends_on: pwm
+    filter: dt_compat_enabled("test-pwm-to-gpio-loopback")
+    harness: ztest
+    harness_config:
+      fixture: gpio_loopback
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
```

Comment:
Can it be also run at nrf54l15?
and for PPR and FLPR?

Reply:
Test can't be run for FLPR right now because of https://github.com/nrfconnect/sdk-zephyr/pull/1911#pullrequestreview-2198703852

Reply:
For PPR case, there is some issue with UICR/nrf-regtool and gpio driver not work right now on PPR